README.md

```diff
@@ -208,27 +182,181 @@ schema = infer(headers, values)
 
 The number of rows used by `infer` can be limited with the `row_limit` argument.
 
-### CLI
+### Validate
+
+Given a schema as JSON file, url to JSON file, or a Python dict, `validate` returns `True` for a valid JSON Table Schema, or raises an exception, `SchemaValidationError`.
+
+```python
+import io
+import json
+
+from jsontableschema import validate
+
+filepath = 'schema_to_validate.json'
+
+with io.open(filepath) as stream:
+    schema = json.load(stream)
+
+is_valid = jsontableschema.validate(schema)
+print(is_valid)
+# True
+
+```
+
+It may be useful to report multiple errors when validating a schema. This can be done with `validator.iter_errors()`.
+
+```python
+
+from jsontableschema import validator
+
+filepath = 'schema_with_multiple_errors.json'
+with io.open(filepath) as stream:
+    schema = json.load(stream)
+    errors = [i for i in validator.iter_errors(schema)]
+```
+
+Note: `validate()` validates whether a **schema** is a validate JSON Table Schema. It does **not** validate data against a schema.
+
+### Export/import
```

Comment:
@roll no: definitely stuff like this as plugins, meaning, install them if you need them. The out-of-the-box thing was more about the `registry` and `validate` stuff, where I finally agreed that @vitorbaptista was right that they are too core to be external modules.